cordapp-cpk/src/test/kotlin/net/corda/plugins/cpk/GradleProject.kt

```diff
@@ -33,7 +34,7 @@ import kotlin.test.fail
 const val expectedCordappContractVersion = 2
 const val expectedCordappWorkflowVersion = 3
 const val expectedCordappServiceVersion = 4
-const val cordaReleaseVersion = "4.6"
+const val cordaReleaseVersion = "4.8"
```

Comment:
Probably very obvious answer, but why is this attribute "4.*" ?

Reply:
Because no "stable" artifacts exist yet for Corda 5.x :grinning:. All I need here is for some `net.corda` artifacts from a real Maven repository to test that the `cordapp-cpk` plugin _refuses_ to use them.

It also helps that the Corda 4.x repository is public...